terraform/locals.tf

```diff
@@ -4,8 +4,6 @@ locals {
   # TLS v1.2, protocols HTTP/1.1 and HTTP/2
   fastly_tls12_sni_configuration_id = "5PXBTa6c01Xoh54ylNwmVA"
 
-  fastly_shield = "iad-va-us"
```

Comment:
Maybe we could have like a "dict" (I dunno what Terraform calls it) for the shields?

I'm imagining something like (maybe invalid syntax but hopefully you get the idea):

```terraform
fastly_shield = {
  cache = "iad-va-us"
  channels = "iad-va-us"
  tarballs = "dub-dublin-ie"
  releases = "dub-dublin-ie"  
}
```

Reply:
I'm not particularly opposed to it but I also don't see the benefit compared to just inlining it in the Fastly service configuration here. There's plenty of service-specific knobs that we're not configuring through such maps right now.